app/models/container_definition.rb

```diff
@@ -6,6 +6,7 @@ class ContainerDefinition < ApplicationRecord
   has_many :container_env_vars,     :dependent => :destroy
   has_one :container,               :dependent => :destroy
   has_one :security_context,        :as => :resource, :dependent => :destroy
+  has_and_belongs_to_many           :container_images, :join_table => :containers
```

Comment:
> has_many :container_images, through: :container

this causes an exception when going through those textual helper methods in [container_summary_helper.rb#L81-81](https://github.com/zeari/manageiq/blob/caaf2e460581d8c52af37099d73c0b1a16c9b9e6/app/helpers/container_summary_helper.rb#L81-81)